.travis.yml

```diff
@@ -9,16 +9,19 @@ jobs:
       stage: test
       compiler: clang
       python: 2.7
+      node_js: 7
       addons:
         apt:
           sources: ['ubuntu-toolchain-r-test']
-          packages: ['cmake', 'nodejs', 'g++-5']
+          packages: ['cmake', 'g++-5']
       before_install:
         - export CC="${CC_COMPILER}"
         - export CXX="${CXX_COMPILER}"
         - export ASAN_OPTIONS="symbolize=1"
       install:
         - pip install --user flake8
+        - "curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -"
```

Comment:
You could just prepend a few lines of code to the js (in which you emit the asm.js and other code to call into it), in which the polyfill is done (see code [here](https://github.com/kripken/emscripten/blob/incoming/src/preamble.js#L1795)).